kartothek/io/dask/dataframe.py

```diff
@@ -25,17 +28,17 @@
 
 @default_docs
 def read_dataset_as_ddf(
-    dataset_uuid=None,
-    store=None,
-    table=None,
-    columns=None,
-    concat_partitions_on_primary_index=False,
-    predicate_pushdown_to_io=True,
-    categoricals=None,
-    label_filter=None,
-    dates_as_object=False,
-    predicates=None,
-    factory=None,
+    dataset_uuid: Optional[str] = None,
+    store: Optional[Callable] = None,
+    table: Optional[str] = None,
+    columns: Optional[List[str]] = None,
+    concat_partitions_on_primary_index: bool = False,
+    predicate_pushdown_to_io: bool = True,
+    categoricals: Optional[List[str]] = None,
+    label_filter: Optional[Callable] = None,
+    dates_as_object: bool = False,
+    predicates: PredicatesType = None,
```

Comment:
I think mypy will complain about this as well: `Optional[...]` is missing for `predicates` and `factory`.

Reply:
We defined `PredicatesType` as `Optional[List[ConjunctionType]]`

Reply:
Therefore `None` would be a valid predicate. We might iterate this but for starters I would like to simply capture what we have